pkgs/development/python-modules/sentence-transformers/default.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  stdenv,
```

Comment:
Sorry, last nit if you don't mind. Can you move `stdenv` just below `lib` please?

Otherwise, LGTM!

Reply:
No worries. Done.